mozdownload/scraper.py

```diff
@@ -51,6 +51,9 @@
                            'win32': 'exe',
                            'win64': 'exe'}
 
+DEFAULT_BRANCHES = {'thunderbird': 'comm-central'
```

Comment:
So I've to add entries for firefox and fennec in DEFAULT_BRANCHES dictionary right?